plugins/meson.build

```diff
@@ -1,4 +1,3 @@
-subdir('altos')
 subdir('dfu')
```

Comment:
`libm `is marked as optional but the dfu plugin is automatically compiled.  Either `dfu `should get a compile option or `libm `should be marked mandatory in toplevel `meson.build`

Reply:
I think the meson optional thing is to allow libm to be a external library like on Solaris rather than a built in thing like Linux. It's a throwback to the 80s.